src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileMeta.java

```diff
@@ -112,8 +120,7 @@ public String getFilePath()
         return filePath;
     }
 
-    public void writeExternal( ObjectOutput out )
-            throws IOException
+    public void writeExternal( ObjectOutput out ) throws IOException
```

Comment:
Let's include blockSize as a field we read/write in these serde methods. If we write a bunch of data and then change the block size, we don't want to render these blocks unreadable.

Reply:
All set - cleaning up serialization was on my list of things todo prior.